assets/airports/ksav.json

```diff
@@ -0,0 +1,1899 @@
+    },
+    "icao": "KSAV",
+    "iata": "SAV",
+    "magnetic_north": -5.5,
+    "ctr_radius": 50,
+    "ctr_ceiling": 10000,
+    "initial_alt": 3000,
+    "position": ["N32d07m39.298", "W81d12m07.675", "10ft"],
+    "rangeRings": {
+        "enabled": true,
+        "center": ["N32d07m39.298", "W81d12m07.675"],
+        "radius_nm": 5.0
+    },
+    "has_terrain": true,
+    "wind": {
+        "angle": 100,
+        "speed": 3
+    },
+    "arrivalRunway": "10",
+    "departureRunway": "10",
+    "defaultMaps": ["Base Map"],
+    "airspace": [
+        {
+            "floor": 0,
+            "ceiling": 100,
+            "airspace_class": "C",
+            "poly": [
+                ["N31d27m32.536", "W80d47m54.451"],
+                ["N31d36m54.489", "W80d40m59.545"],
+                ["N31d59m60.000", "W80d28m59.533"],
+                ["N32d09m26.793", "W80d38m00.780"],
+                ["N32d16m56.773", "W80d40m05.225"],
+                ["N32d16m40.446", "W80d49m00.393"],
+                ["N32d19m56.757", "W80d55m59.531"],
+                ["N32d29m53.114", "W81d01m33.275"],
+                ["N32d38m00.659", "W81d01m23.921"],
+                ["N32d38m00.473", "W81d27m44.239"],
+                ["N32d32m00.423", "W81d33m13.793"],
+                ["N32d33m16.359", "W81d41m07.370"],
+                ["N32d33m49.489", "W81d43m04.226"],
+                ["N32d33m49.487", "W81d45m28.993"],
+                ["N32d32m39.052", "W81d48m10.860"],
+                ["N32d30m06.503", "W81d49m59.745"],
+                ["N32d27m21.822", "W81d49m59.755"],
+                ["N32d19m43.966", "W81d43m55.892"],
+                ["N32d01m27.257", "W81d59m59.984"],
+                ["N31d50m59.392", "W81d59m59.788"],
+                ["N31d35m26.863", "W81d47m53.413"],
+                ["N31d35m26.382", "W81d28m01.305"],
+                ["N31d33m26.671", "W81d26m02.468"],
+                ["N31d27m32.555", "W81d26m02.621"]
+            ]
+        }
+    ],
+    "fixes": {
+        "ANKOR" : ["N31d51m33.995", "W81d12m13.622"],
+        "BASLE" : ["N32d09m43.362", "W80d45m48.490"],
+        "BROSE" : ["N32d17m12.687", "W81d40m56.549"],
+        "CATHY" : ["N31d40m11.624", "W81d10m17.264"],
+        "CESOR":  ["N32d36m13.089", "W81d38m32.314"],
+        "CINBA":  ["N32d17m06.491", "W80d58m40.208"],
+        "ESENT":  ["N32d09m28.130", "W80d55m03.779"],
+        "FAPLI":  ["N32d15m12.715", "W81d20m58.781"],
+        "FILOX":  ["N32d19m42.713", "W81d38m27.873"],
+        "FRIES":  ["N32d13m52.696", "W80d47m37.956"],
+        "GIPPL":  ["N31d22m53.957", "W81d09m53.696"],
+        "HARPS":  ["N31d41m29.560", "W81d15m15.735"],
+        "HENIG":  ["N32d10m17.993", "W80d48m13.464"],
+        "JAMEB":  ["N32d22m10.265", "W81d58m36.549"],
+        "JANIE":  ["N31d51m33.930", "W81d17m44.602"],
+        "JROSS":  ["N32d42m40.008", "W80d37m37.990"],
+        "KUCES":  ["N32d42m40.008", "W80d37m37.990"],
+        "KSAV":   ["N32d07m39.298", "W81d12m07.675", "Savannah"],
```

Comment:
This isn't needed, see my comment about the spawnPatterns

Reply:
Fixed